src/main/java/nl/tudelft/ewi/sorcerers/AppConfig.java

```diff
@@ -43,7 +46,7 @@
 	@Inject

```

Comment:
*Auto-generated comment*

    File contains tab characters (this is the first instance).